### PR TITLE
Safe remote dir creation

### PR DESCRIPTION
**Pull request by realsystem**

### CODE REVIEW

**Comment by Wasapl** [comment]
Build finished.
Unittest failed

**Comment by Wasapl** [comment]
Unittest failed

**Comment by Wasapl** [comment]
Build finished.
Functional test success

**Comment by Wasapl** [comment]
Functional test success

**Comment by Wasapl** [comment]
Build finished.
Unittest failed

**Comment by Wasapl** [comment]
Unittest failed

**Comment by Wasapl** [comment]
Build finished.
Unittest success

**Comment by Wasapl** [comment]
Unittest success

**Comment by Wasapl** [comment]
Build finished.
Unittest success

**Comment by Wasapl** [comment]
Unittest success

**Comment by Wasapl** [comment]
Build finished.
Functional test success

**Comment by Wasapl** [comment]
Functional test success

**Comment by miarmak** [comment]
Looks good, +1.

File: cloudferrylib/utils/files.py [hunk 1]
```diff
@@ -52,5 +53,24 @@ def __exit__(self, exc_type, exc_val, exc_tb):
...
+        self.runner.run_ignoring_errors(_unlink_dir(self.dirname))
+        return self
+
+
 def _unlink(filename):
     return "rm -f {file}".format(file=filename)
+
+
+def _unlink_dir(dirname):
+    return "rm -rf {dir}".format(dir=dirname)
```
Comment (roman-verchikov) [inline cloudferrylib/utils/files.py]: maybe `rmdir`?

**Comment by roman-verchikov** [comment]
+1, functional tests succeed. Can be merged

**Pull request merged**